jkube-kit/generator/java-exec/src/test/java/org/eclipse/jkube/generator/javaexec/JavaExecGeneratorTest.java

```diff
@@ -111,9 +111,9 @@ void createAssemblyWithNoFatJarShouldAddDefaultFileSets() {
     // Then
     assertThat(result)
         .hasFieldOrPropertyWithValue("excludeFinalOutputArtifact", false)
-        .extracting(AssemblyConfiguration::getLayers).asList().hasSize(1)
+        .extracting(AssemblyConfiguration::getLayers).asInstanceOf(InstanceOfAssertFactories.list(Assembly.class)).hasSize(1)
         .first().asInstanceOf(InstanceOfAssertFactories.type(Assembly.class))
-        .extracting(Assembly::getFileSets).asList()
+        .extracting(Assembly::getFileSets).asInstanceOf(InstanceOfAssertFactories.list(Assembly.class))
```

Comment:
Shouldn't we be using AssemblyFileSet here?

Reply:
Thanks @rohanKanojia I missed it. I have pushed the change.